src/java.desktop/share/classes/javax/swing/plaf/basic/BasicFileChooserUI.java

```diff
@@ -724,7 +724,7 @@ public void valueChanged(ListSelectionEvent evt) {
         // class calls into the Handler.
         Handler handler;
         /**
-         * Constucts a {@code DoubleClickListener}.
+         * Constructs a {@code DoubleClickListener}.
```

Comment:
This change is under review here:
https://github.com/openjdk/jdk/pull/7030

Reply:
Oops, I'll remove mine. Thanks.